dataprep-pipelines/Cargo.toml

```diff
@@ -7,6 +7,7 @@ license.workspace = true
 repository.workspace = true
 
 [dependencies]
+dir-assert = "0.2.0"
```

Comment:
use the workspace version

Reply:
@thea-exe

Reply:
say dir-assert.workspace = true
and then leave the vresion specified in the top level workspace cargo.toml